apps/api/src/server/migration/migrators/nsip-subscription-migrator.js

```diff
@@ -142,15 +141,13 @@ const mapModelToEntity = async (m) => {
 		throw Error('Missing subscription ID');
 	}
 
-	const entity = {
+	const entity = typesToSubscription(m.subscriptionTypes, {
```

Comment:
This is the key bit, `typesToSubscription` no longer alters the object passed in, it returns new one.